packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/interfaces.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Nice. I think we shoudl stick with this interfaces.ts convention.

Reply:
I was kind of torn between making this `types.ts` instead but Julian mentioned recently we should just use interfaces for everything where we can and I don't have a strong opinion on it so yeah, SGTM.